mycroft/skills/media/vocab/en-us/PrevKeyword.voc

```diff
@@ -0,0 +1 @@
+play previous
```

Comment:
It's better to follow similar commands for Next and Prev in order to help the user to easily understand the feature.
So, why not add `previous track` too?